src/main/webapp/scripts/toggle.js

```diff
@@ -70,12 +75,20 @@ function hideLoading() {
   $("#real-body").removeClass("hide").addClass("body");
 }
 
+function makeUrl(url, keyword) {
+  let queryString = `?key=${keyword}`;
+  let fullUrl = `${url}${queryString}`;
+  if (url === '/news' && locationObj != null) {
+    fullUrl = `${fullUrl}&country=${locationObj["Short Country"]}`;
```

Comment:
[Optional] Using dot notation instead of []. This would require removing the space from 'Short Country'

Reply:
Would 'Short_Country' be the most stylistically consistent?